static/js/fx-bento.js

```diff
@@ -2,10 +2,11 @@
 
 function getFxAppLinkInfo(localizedBentoStrings, referringSiteURL) {
   return [
-    [localizedBentoStrings.fxMonitor, "https://monitor.firefox.com/", "fx-monitor"],
+    [localizedBentoStrings.fxMonitor, "https://monitor.firefox.com/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=nav&amp;utm_content=products", "fx-monitor"],
```

Comment:
We'll probably want to align this with the other UTM parameters?

```suggestion
    [localizedBentoStrings.fxMonitor, "https://monitor.firefox.com/?utm_source=${referringSiteURL}&amp;utm_medium=referral&amp;utm_campaign=bento&amp;utm_content=desktop", "fx-monitor"],
```

Reply:
Ah yes, good call